layouts/partials/default.html

```diff
@@ -22,7 +22,27 @@
           {{ partial "breadcrumb.html" . }}
         </div>
         <!-- end breadcrumb feature -->
-        <div class="p-lg-5 p-4 bg-white" id="content">
+
+        <!-- begin "Edit This Page" button -->
+        <div class="p-lg-5 p-4 bg-white position-relative" id="content">
+          <div class="d-flex flex-row-reverse">
+            {{ $path := "" }}
+            {{ with .File }}
+                {{ $path = .Path }}
+                {{ else }}
+                {{ $path = .Path }}
+            {{ end }}
+            {{ if eq .Params.hide_edit_btn true }}
+                {{ else }}
+                <a href="{{ .Site.Params.editPrefix}}/{{ $path }}">
+                  <button class="editBtn py-2 px-3"><span>Edit this Page
+                    </span></button>
```

Comment:
Seems like you will need to wrap the `<a>` element inside of the `<button>`.

Reply:
I did this to make the whole button a link. May be this isn't the good practice. So,I used a form to make the button a link which redirects to the desired page.